src/main/webapp/app/shared/pages/imprint-page/imprint-page.component.scss

```
@@ -0,0 +1,35 @@
+.imprint-page {
+  color: var(--p-text-primary);
+  padding: 3rem 5rem 1.5rem 1.5rem;
+  max-width: 1200px;
+  margin-right: auto;
+  .headline {
+    font-size: 2.3rem;
```

Could you please change this to 2rem so it matches the headlines of the other pages in the website 
```suggestion
    font-size: 2rem;
```